training/src/test/java/wolox/training/controllers/BookControllerTest.java

```diff
@@ -282,6 +301,13 @@ void whenFindFirstByIsbnInService_thenReturnABook() throws Exception {
         Mockito.when(mockBookRepository.save(Mockito.any()))
             .thenReturn(oneTestBookWithIsbn);
 
+        stubFor(WireMock.get(urlEqualTo(
+            RESQUEST_ISBN_BOOK))
+            .willReturn(aResponse()
+                .withStatus(HttpStatus.OK.value())
+                .withHeader("Content-Type", "application/json")
+                .withBody(RESPONSE_ISBN_BOOK)));
```

Comment:
Just so you know, I think you could put the response on a json file and use `withBodyFile` instead of having the mocked response as a json in a java variable.